tool.c

```diff
@@ -221,9 +222,105 @@ compdb(int argc, char *argv[])
 	return 0;
 }
 
+static void
+targets_rule(int argc, char *argv[])
+{
+	struct edge *e = NULL;
```

Comment:
No need to initialize this to `NULL`, also please add a blank line separating the initial declarations.

Reply:
Done